code/modules/mob/living/carbon/human/life.dm

```diff
@@ -368,20 +368,35 @@ All effects don't start immediately, but rather get worse over time; the rate is
 81-90: Extremely high alcohol content - light brain damage, passing out
 91-100: Dangerously toxic - swift death
 */
-
+#define BALLMER_POINTS 5
 /mob/living/carbon/human/handle_status_effects()
 	..()
+
+
 	if(drunkenness)
 		drunkenness = max(drunkenness - (drunkenness * 0.04), 0)
-
 		if(drunkenness >= 6)
 			if(prob(25))
 				slurring += 2
 			jitteriness = max(jitteriness - 3, 0)
 
 		if(drunkenness >= 11 && slurring < 5)
 			slurring += 1.2
-
+		if(mind && (mind.assigned_role == "Scientist" || mind.assigned_role == "Research Director"))
+			if(SSresearch.science_tech)
+				if(drunkenness >= 12.9 && drunkenness <= 13.8)
+					var/ballmer_percent = 0
+					if(drunkenness == 13.35) // why run math if I dont have to
```

Comment:
A == comparison on a floating point number?  There's a significant chance that that will LITERALLY never trigger.  Make it a range.

Reply:
That's intentional.

Reply:
With this few significant digits it'll trigger just fine even as a floating point number. As I said in my above comment though it should definitely round drunkenness.